rcsb/utils/insilico3d/ModelReorganizer.py

```diff
@@ -287,6 +306,58 @@ def __rebuildEntryIds(self, dataContainer, sourceModelEntryId, sourceModelDb, in
 
         return dataContainer
 
+    def __rebuildDateDetails(self, dataContainer, sourceModelEntryId, sourceArchiveReleaseDate, sourceArchiveModifiedDate):
+        """Add or rebuild release and revision date details for the dataContainer.
+
+        Mainly for ModelArchive models which currently lack this information in the mmCIF file (as of 19-Oct-2022 dwp).
+
+        Args:
+            dataContainer (object): mmcif.api.DataContainer object instance
+            sourceModelEntryId (str): source entry ID
+            sourceArchiveReleaseDate (str): release date for dataContainer, obtained from source model website (e.g., '2022-09-28')
+            sourceArchiveModifiedDate (str): last-modified date for dataContainer, obtained from source model website (e.g., '2022-09-28T17:22:53.310750Z')
+
+        Returns:
+            dataContainer: updated dataContainer object
+        """
+        # If pdbx_database_status.* doesn't exist, create it and add release date
+        if not dataContainer.exists("pdbx_database_status"):
+            dataContainer.append(
+                DataCategory(
+                    "pdbx_database_status",
+                    attributeNameList=["entry_id", "recvd_initial_deposition_date", "status_code"],
```

Comment:
@piehld We know the release date from the website. Is the deposition date included on the website? If not, we should not include it.

Reply:
Yes, the deposition date is provided via their API used to populate the page: https://www.modelarchive.org/api/projects/ma-bak-cepc (see the key `"release_date"`)

Reply:
And see `"modified_at"` for the last modified date

Reply:
> @piehld We know the release date from the website. Is the deposition date included on the website? 
Oh, I see the point you're trying to make—the difference between the meaning of "deposition" vs "release" date. I guess "deposition" is not included explicitly, so I can exclude that. It shouldn't be a problem right?

Reply:
@piehld It is not clear to me why the `release_date` same that the deposition date. What was modified on the `modified_at` date?

Reply:
> > @piehld We know the release date from the website. Is the deposition date included on the website?
> 
> Oh, I see the point you're trying to make—the difference between the meaning of "deposition" vs "release" date. I guess "deposition" is not included explicitly, so I can exclude that. It shouldn't be a problem right?

`_pdbx_database_status.recvd_initial_deposition_date` is not mandatory. It can be excluded or set to `?`.

Reply:
We need `_pdbx_database_status.entry_id` and `_pdbx_database_status.status_code`. `_pdbx_database_status.recvd_initial_deposition_date` is not mandatory and is not available in this case.

Reply:
OK thanks. I'll leave `_pdbx_database_status.entry_id` and `_pdbx_database_status.status_code` in there then. I guess we were populating those at some other point in the pipeline then.

Reply:
Yes, these are populated during the ExchangeDB ETL process, if they are not available in the mmCIF file. But if we are adding the other audit records to the mmCIF file, then we might as well do this here to keep it simple.

Reply:
Agreed

Reply:
> @piehld It is not clear to me why the `release_date` is the same as the deposition date. What was modified on the `modified_at` date?

@brindakv I'm not exactly sure, but if you look at https://www.modelarchive.org/api/projects/ma-bak-cepc, it says:
```
"modified_at": "2022-08-29T15:46:03.193830Z",
"modified_by": {
"id": 302,
"email": "ianhumph@uw.edu",
"first_name": "Ian",
"last_name": "Humphreys",
"is_admin": false
},
```

Reply:
Without knowing what was modified, we cannot use this information. `_pdbx_audit_revision_details.type` is mandatory and cannot be populated without knowing what was modified.

Reply:
OK, that's understandable. I'll remove it. Should have probably conferred with you before getting all excited and adding it in here everywhere...